src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/BuildIntegrationTest.cs

```diff
@@ -625,13 +625,13 @@ public async Task Build_WithoutServer_ErrorDuringBuild_DisplaysErrorInMsBuildOut
         {
             var result = await DotnetMSBuild(
                 "Build",
-                "/p:UseRazorBuildServer=false /p:RazorLangVersion=5.0",
+                "/p:UseRazorBuildServer=false /p:RazorLangVersion=6.0",
```

Comment:
Minor point, but why keep rolling this forward each time? What about:

```suggestion
                "/p:UseRazorBuildServer=false /p:RazorLangVersion=99999.0",
```

Reply:
I don't really know why this test is authored this way. @NTaylorMullen \ @ajaybhargavb ?

Reply:
No reason. I was probably being dumb.